persistence/src/main/scala/hmda/persistence/processing/HmdaFileValidator.scala

```diff
@@ -209,7 +209,7 @@ class HmdaFileValidator(submissionId: SubmissionId) extends HmdaPersistentActor
       self ! CompleteValidation(replyTo)
 
     case CompleteValidation(replyTo) =>
-      if (state.larSyntactical.isEmpty && state.larValidity.isEmpty && state.larQuality.isEmpty && state.larMacro.isEmpty
+      if (state.larSyntactical.isEmpty && state.larValidity.isEmpty && state.qualityVerified && state.macroVerified
         && state.tsSyntactical.isEmpty && state.tsValidity.isEmpty && state.tsQuality.isEmpty) {
```

Comment:
Looks like `&& state.tsQuality.isEmpty` was left in by accident. That shouldn't be a check in this conditional.

Also, we now have `validityErrors` and `syntacticalErrors` convenience methods on the `HmdaFileValidationState`, so you could use those to simplify this expression if that would help readability.